find_email_addresses.py

```diff
@@ -0,0 +1,45 @@
+import sys, re
+import link_extractor as le
+from collections import deque
+
+debug = False
+
+try:
+	domain = sys.argv[1]
+except:
+	domain = 'web.mit.edu'
```

Comment:
i might lean towards providing output to the user if the # of arguments was incorrect, but i like that you included a default value.